meta-refkit/conf/distro/include/refkit-development.inc

```diff
@@ -33,3 +33,4 @@ REFKIT_IMAGE_BUILD_MODE_SELECTED = "1"
 
 REFKIT_IMAGE_COMPUTERVISION_EXTRA_FEATURES_append = " computervision-test"
 REFKIT_IMAGE_FEATURES_COMMON_append = " common-test"
+BAD_RECOMMENDATIONS_remove = "bluez5-client"
```

Comment:
I rebased the commit and amended it to only include the BAD_RECOMMENDATIONS_remove entry.